### ISSUE TITLE
Overview issue: Leaderboard 2.0 release 

### ISSUE THREAD

**Issue by KennethEnevoldsen** [embeddings-benchmark/mteb#1405]
- [x] #1372 
- [x] https://github.com/embeddings-benchmark/mteb/issues/1396
- [x] Ensure old results are present in the new leaderboard (https://github.com/embeddings-benchmark/results/pull/43)
- [ ] Review of a public demo of the leaderboard
- [x] Add new results https://github.com/embeddings-benchmark/results/pull/40
- [x] https://github.com/embeddings-benchmark/mteb/issues/1373
- [x] https://github.com/embeddings-benchmark/mteb/issues/1317 (Solved with hotfixes, waiting for Gradio)
- [x] #1439 

Once everything is ready I will push the preprint of the paper and we can make a coordinated public release.

**Comment by isaac-chung** [comment]
Added #1317 and #1373 as well. Feel free to remove / reclassify them if they are nice-to-haves instead of need-to-haves.

**Comment by x-tabdeveloping** [comment]
We now have a [demo of the new leaderboard](https://huggingface.co/spaces/mteb/leaderboard_2_demo). Feel free to either open new issues as they come along or conduct discussions in this thread

**Comment by SaitejaUtpala** [comment]
![image](https://github.com/user-attachments/assets/e2a6a211-2dad-4361-b53e-7fdb0bf6ae24)
new leaderboard seems to be down.

**Comment by x-tabdeveloping** [comment]
I believe it's due to an issue in Arctic models' metadata, I'll have a look

**Comment by x-tabdeveloping** [comment]
I have found the source of the problem, working on a fix rn.

**Comment by x-tabdeveloping** [comment]
Fixed

**Comment by KennethEnevoldsen** [comment]
@gaohongkui note sure what you mean?

**Comment by gaohongkui** [comment]
> @gaohongkui note sure what you mean?

I mean, if the metric of task_type can be switched at a certain position on the page, 
and then the leaderboard is sorted according to the new metric (for example, using recall@10 or ndcg@10 for Retrieval may result in different sorting results). 

This is convenient for observing the model effect from more angles.

This might be considered a feature request. Should I discuss it here or open a separate issue?

**Comment by KennethEnevoldsen** [comment]
Ahh gotcha, feel free to open a seperate issue on this. Probably won't be in the first release of the leaderboard though

**Comment by KennethEnevoldsen** [comment]
**Overview of what is missing**

We opened the following issues during review:

- [ ] https://github.com/embeddings-benchmark/mteb/issues/1411
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1509
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1571
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1514
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1516
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1550
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1556
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1605

Nice to have, but not need to have:

- [ ] https://github.com/embeddings-benchmark/mteb/issues/1013
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1622
- [ ] https://github.com/embeddings-benchmark/mteb/issues/1517

**Comment by isaac-chung** [comment]
We have 2 leaderboard overview issues with https://github.com/embeddings-benchmark/mteb/issues/1867. Which list should we follow such that we're happy with the release (with the must-haves only)?

**Comment by KennethEnevoldsen** [comment]
I will close this in favor of the other - all the required PRs here have been resolved
